src/OpenTelemetry.Extensions.Enrichment/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenTelemetry .NET SDK telemetry enrichment framework
+
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.Enrichment.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Enrichment)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.Enrichment.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Enrichment)
+
+Contains OpenTelemetry .NET SDK telemetry enrichment framework
+which is used for enrichment of logs, metrics, and traces.
```

Comment:
nit: Why say `metrics` and `logs`? It is only traces currently, we can always add to this when other signals are supported 😄